content/may2015/authorization.md

```diff
@@ -93,54 +94,111 @@ The [bearer access token][7] returned from the authorization server is what you
 
 ```
-{ 
+{
   "access_token" : "18adCadfj_lj13S3ada8.41jVCo_dgalL",
   "encounter" : "91731344",
   "patient" : "121341578",
   "id_token" : "eyAiYWxnIjoibm9uZSIKfQ==.ewogICJpc3MiOiAiaHR0cHM6Ly9hdXRob3JpemF0aW9uLmRldmNlcm5lci5jb20vb2F1dGgyIiwKICAic3ViIjogIm1yNTE0QGNlcm5lci5jb20iLAogICJhdWQiOiAiY2xpZW50X2FwcCIsCiAgIm5vbmNlIjogIm4tMFM2X1d6QTJNaiIsCiAgImV4cCI6IDEzMTEyODE5NzAsCiAgImlhdCI6IDEzMTEyODA5NzAsCiAgInByb2ZpbGUiOiJodHRwczovL2V4YW1wbGUuY29tLzEyMzQiCn0="
 }
 ```
 
-
 ### Using an access token ###
 In order to use an access token, your client application needs to provide the access token received from the authorization server to the protected resource. The following is a non-normative example of the usage of the token to access a protected RESTful web service on a resource server.  It will need to be added as an authorization HTTP header.
 
 ```
 Authorization: Bearer {ACCESS_TOKEN}
 ```
 
-If the access token is valid, and your application is authorized, the resource server will allow your client application to access its protected resources.
+If the access token is valid and your application is authorized, the resource server will allow your client application to access its protected resources.
 
 ### Using a Refresh Token ###
-The authorization server has support for **online access** refresh tokens. Refresh tokens **must** be used in conjunction with the **online_access** scope. In order to use a refresh token, the user's session **must** remain active. To request a refresh token, **online_access** must be added to the scope query parameter when requesting an authorization code. If this scope is not added, a refresh token will not be returned. Following the [OAuth 2.0][1] spec , the authorization server accepts a content type of **application/x-www-form-urlencoded** **POST** from your client application with the following information.
+The authorization server has support for **online access** refresh tokens. Refresh tokens **must** be used in conjunction with the **online_access** scope. In order to use a refresh token, the user's session **must** remain active. To request a refresh token, **online_access** must be added to the scope query parameter when requesting an authorization code. If this scope is not added, a refresh token will not be returned. Following the [OAuth 2.0][1] spec, the authorization server accepts a content type of **application/x-www-form-urlencoded** **POST** from your client application with the following information.
 
-* grant_type
+* grant_type - "refresh_token"
 * refresh_token
 
 ```
 grant_type=refresh_token&refresh_token={REFRESH_TOKEN}
 ```
 
-If successful, the authorization server will return a **200 OK** response with a content-type of [**application/json**][3] similar to the example below. 
+If successful, the authorization server will return a **200 OK** response with a content-type of [**application/json**][3] similar to the non-normative example below.
 
 ```
```

Comment:
We could change the code fencing to \<pre class="body-response">\<code class="language-javascript"> and \</code>\</pre> to prevent these token examples from displaying on a single massive line.

Also noticed that 'non-normative' was added here but not in DSTU2